driver/src/test/scala/za/co/absa/hyperdrive/driver/drivers/KafkaToParquetDockerTest.scala

```diff
@@ -122,7 +123,6 @@ class KafkaToParquetDockerTest extends FlatSpec with Matchers with SparkTestBase
 
   after {
     SchemaManager.reset()
-    fs.delete(new Path(baseDir), true)
```

Comment:
Why dont you delete after tests ?

Reply:
It's done in the `TempDirectory.deleteOnExit` implementation